<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyloaderDialog</class>
 <widget class="QDialog" name="KeyloaderDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>975</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>975</width>
    <height>550</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tait Keyloader</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>501</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>KeyID (Hex)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtKey">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>KeyID (Hex)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Cipher</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbKeyType">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="butAddKey">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Add to List</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>CKR</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnCKR">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4095</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnKeyID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhPreferUppercase</set>
    </property>
    <property name="prefix">
     <string>0x</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QTableView" name="tblNewKeys">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>471</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>65</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="butSend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Send List to Radio</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="butClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Clear List</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>461</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="butEraseAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red</string>
    </property>
    <property name="text">
     <string>Erase All Keys</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butGetInventory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>441</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Get Loaded Keys</string>
    </property>
   </widget>
   <widget class="QTableView" name="tblInventory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>971</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Ready...</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
